src/model/documentfragment.js

```diff
@@ -84,6 +84,16 @@ export default class DocumentFragment {
 	}
 
 	/**
+	 * Artificial parent of `DocumentFragment`. Returns `null`. Added for compatibility reasons.
```

Comment:
I've just checked that the native doc frag also has a parrentNode and parentElement properties, so it makes some sense here.